tests/test_client.py

```diff
@@ -6,18 +9,21 @@ def test_instantiate(index_client):
         size=5,
         urls=[]
     )
-    assert doc.size == 5
-    assert doc.hashes == hashes
+    if doc.size == 5 or doc.hashes == hashes:
```

Comment:
why turn two lines into 5?

Reply:
assert doc.size == 5 or doc.hashes == hashes if you want one or the other? -- but why?